seed/models/columns.py

```diff
@@ -985,11 +985,12 @@ def select_col_obj(column_name, table_name, organization_column):
             try:
                 # the from column is the field in the import file, thus the table_name needs to be
                 # blank. Eventually need to handle passing in import_file_id
-                from_org_col, _ = Column.objects.get_or_create(
+                from_org_col, _ = Column.objects.update_or_create(
                     organization=organization,
-                    table_name__in=[None, ''],
+                    table_name='',
```

Comment:
This has been updated here and below for two reasons.
- There exists a constraint that doesn't allow `table_name` to be None (null in PG). 
- it seems that `update_or_create` doesn't take `__in`.